src/openapi.yaml

```diff
@@ -60,6 +60,12 @@ paths:
     $ref: './resources/ApplicationDeploy.yaml'
   /application/{applicationId}/event:
     $ref: './resources/ApplicationEvent.yaml'
+  /application/{applicationId}/event?tail=:
+    $ref: './resources/ApplicationEventTail.yaml'
+  /application/{applicationId}/event?lastId=:
```

Comment:
Can't we create only one URL for `/application/:id/event` ? lastId and tail are optional parameters. When not specified, it's the option `tail=50`that is used.

Reply:
as seen together, it was not possible because the response is not the same depending on the query params you use